lib/hako/schedulers/ecs_autoscaling.rb

```diff
@@ -139,23 +165,50 @@ def service_resource_id(service)
         "service/#{service.cluster_arn.slice(%r{[^/]+\z}, 0)}/#{service.service_name}"
       end
 
+      # @return [String]
+      def target_group_resource_label
+        target_group = @ecs_elb_client.describe_target_group
+        load_balancer_arn = target_group.load_balancer_arns[0]
+        target_group_arn = target_group.target_group_arn
+        "#{load_balancer_arn.slice(%r{:loadbalancer/(.+)\z}, 1)}/#{target_group_arn.slice(/[^:]+\z/)}"
```

Comment:
I think validation on ELB type is needed.
Its type must be "application" (must not be "network").

Reply:
Ahh, you're right. Thank you for pointing it out. 7e18ef9